experiments/dataSources/SalesforceREST/EinsteinAnalytics/readme.md

```diff
@@ -0,0 +1,95 @@
+# Show Salesforce Einstein Analytics data in Skuid
+Einstein Analytics (EA) is a powerful data integration and analysis platform.  It does a lot of things Skuid cannot do.  But there are going to be times when you would like to incorporate an EA report into your Skuid page and use Skuid components to show the data - rather than use an Einstein Dashboard or component.  Here is how. 
+
+Some resources:
+
+- [Get a free Salesforce developer org](https://developer.salesforce.com/promotions/orgs/analytics-de) - with EA enabled and sample datasets (that we will be using in these pages)
+- [Ananlytics API doc](https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_rest_resources_overview.htm) (from Salesforce Developer resources):  
+- [Analytics query language (SAQL) reference](https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_saql.meta/bi_dev_guide_saql/bi_saql_intro.htm)
+
+
+## How to use this data source
+
+### Create the data source
+
+Follow the instructions in the "SalesforceREST" folder immediately above to create a generic "Salesforce REST" datasource. 
+
+### Create a new page (V2 page API)  and copy the XML from the file:  "Einstein_Analytics_Resources"  
+
+This will show you all the datasets and lenses you have enabled in the org and give you direct links to them. 
+
+## But I want some Data! 
+
+Showing Data from Einstein Analytics requires use of the [Query Endpoint](https://developer.salesforce.com/docs/atlas.en-us.bi_dev_guide_rest.meta/bi_dev_guide_rest/bi_resources_query.htm)
+
+Here is how: 
+
+#### Find a resource in Einstein Analytics studio - prepare it for use in a query
+
+-   Open a lens or dashboard element.  Or create a new one! 
+-   Open the "Query mode" view of a lens,  or use the "more details" button on the details panel of a dashboard.  Look for the query code.  It looks like this: 
+
+```
+q = load "DTC_Opportunity_SAMPLE";
+q = group q by ('Industry', 'Product_Family');
+q = foreach q generate 'Industry' as 'Industry', 'Product_Family' as 'Product_Family', sum('Amount') as 'sum_Amount';
+q = order q by 'sum_Amount' desc;
+q = limit q 2000; 
+```
+    
+- Copy this code to a text editor and make the following changes: 
+    -  Add a JSON wrapper to the query.  
+        - Append Append ``{"query":"``  to the beginning
+        - Finish block with ``"}``
+
+    - Replace the name of the dataset (that comes in the first line after “load” with a combination of its id and current version id.  You can find the Id’s in the EA_Resources page that we created above. 
```

Comment:
We don't close the parentheses here, so switch it to a comma? Also take out the apostrophe.

```suggestion
    - Replace the name of the dataset, which comes in the first line after “load” with a combination of its id and current version id.  You can find the Ids in the EA_Resources page that we created above. 
```